### ISSUE TITLE
Add labels to each file in dataset

### ISSUE THREAD

**Issue by evansavage** [DDMAL/xml-parse-glyph-img#6]
This could be done with a text file, or by ordering into parallel folders based on position and neume tag respectively

**Comment by vigliensoni** [comment]
What is the format that Alicia's code is expecting? That may inform the decision we take.

**Comment by vigliensoni** [comment]
@evansavage, any updates on this?

**Comment by evansavage** [comment]
Alicia's code is expecting a dataset comprised of a few entries:

* **Image Shape**: bounding box of glyph
* **Image Position**: bounding box of glyph in staff, same width as "Image Shape," but greater height
* **Shape**: Text label referring to glyph type 
* **Position**: Text label for position in staff
* **Symbol**: Concatenation of "Shape" and "Position"

Her code is also parsing the individual images for each of the glyphs based on JSON files that are loaded in parallel with the images of the scores. I plan on writing a new CoLab script, borrowing elements from Alicia's code, since we are concerned with position currently. The tutorials I've followed via Udacity also worked with a more Tensorflow-based approach, which Keras is built upon. I plan to integrate these two workflows for our augmented code.
